tests/test_mission.py

```diff
@@ -506,7 +506,8 @@ def test_radio_channels(self):
         unit.set_client()
         frequency = 300.0
         self.assertNotAlmostEqual(unit.radio[2]["channels"][1], frequency)
-        unit.set_radio_channel_preset(2, 1, frequency)
```

Comment:
Do we really need to fix the test here?

Reply:
Not really. My thinking was that if someone changes the type for the test to an A-10 it will still work,

Also, if the test code sometimes is used as an example for other code, maybe it's good to show that you need to check for `radio` before setting channel presets?